params/copy_params.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.567
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Kanyoni kaja"
```

Comment:
I think there is an extra space before `Kanyoni` here, would you be able to remove it please @gachiethomas ?

```suggestion
species.name <- "Kanyoni kaja"
```